From f4b5d049dafa59b2e683c6dc0b74ae353e530b5a Mon Sep 17 00:00:00 2001
From: eng33 <eng33@waveshare.com>
Date: Wed, 3 Jan 2024 11:46:50 +0800
Subject: [PATCH 0839/1085]  drivers/gpu/drm/panel:Modified the timing of
 11.9inch to fix the issue that 11.9inch was displayed abnormally

Signed-off-by: eng33 <eng33@waveshare.com>
---
 drivers/gpu/drm/panel/panel-waveshare-dsi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/panel/panel-waveshare-dsi.c
+++ b/drivers/gpu/drm/panel/panel-waveshare-dsi.c
@@ -131,7 +131,7 @@ static const struct drm_display_mode ws_
 	.hdisplay = 320,
 	.hsync_start = 320 + 60,
 	.hsync_end = 320 + 60 + 60,
-	.htotal = 320 + 60 + 60 + 120,
+	.htotal = 320 + 60 + 60 + 60,
 	.vdisplay = 1480,
 	.vsync_start = 1480 + 60,
 	.vsync_end = 1480 + 60 + 60,
